packages/engine-rn-tvos/src/__tests__/tasks.test.ts

```diff
@@ -0,0 +1,65 @@
+import { createRnvApi, createRnvContext, getContext } from '@rnv/core';
+import taskRnvRun from '../tasks/task.rnv.run';
+import taskRnvStart from '../tasks/task.rnv.start';
+
+jest.mock('fs');
+jest.mock('axios');
+jest.mock('@rnv/core');
+jest.mock('@rnv/sdk-apple');
+
+beforeEach(() => {
+    createRnvContext({ program: { platform: 'tvos' } });
+    createRnvApi();
+});
+
+afterEach(() => {
+    //Do nothing
+});
+
+const originTask = undefined;
+
+const { executeAsync, logError } = require('@rnv/core');
+
+test('Execute task.rnv.run', async () => {
+    // GIVEN
+    const ctx = getContext();
+    executeAsync.mockReturnValue(Promise.resolve('{}'));
+    // WHEN
+    await taskRnvRun.fn(ctx, undefined, originTask);
+    // THEN
+    await expect(taskRnvRun.fn(ctx, undefined, originTask)).resolves.toEqual(true);
+    // expect(taskManager.executeTask).toHaveBeenCalledWith(c, 'project configure', 'platform list', originTask);
+});
+
+test('Execute task.rnv.start with no parent', async () => {
+    // GIVEN
+    const ctx = getContext();
+    executeAsync.mockReturnValue(Promise.resolve('{}'));
+    // WHEN
+    await taskRnvStart.fn(ctx, undefined, originTask);
+    // THEN
+    await expect(taskRnvRun.fn(ctx, undefined, originTask)).resolves.toEqual(true);
```

Comment:
@mihaiblaga89 is this really meant to be here? you are testing taskRnvStart but then also calling up taskRnvRun. same below

Reply:
good catch, mistake. Will fix